elasticsearch/examples/upgrade/scripts/upgrade.sh

```diff
@@ -43,10 +43,17 @@ do
   esac
 done
 
+if ! command -v jq > /dev/null
+then
+  echo 'jq is required to use this script'
+  echo 'please check https://stedolan.github.io/jq/download/ to install it'
+  exit 1
+fi
+
 # Elasticsearch chart < 7.4.0 are not compatible with K8S >= 1.16)
 if [[ -z $FROM ]]
 then
-  KUBE_MINOR_VERSION=$(kubectl version --client -o yaml | grep minor | sed 's/[^0-9]*//g')
+  KUBE_MINOR_VERSION=$(kubectl version -o json | jq -c '.serverVersion.minor' | sed 's/[^0-9]*//g')
```

Comment:
For `jq` I think `--raw-output` could make more sense than `-c`, and we should add `--exit-status` so it'll actually break if it doesn't find the property.